django/publicmapping/generate_settings.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+from optparse import OptionParser
+import sys
+from redistricting import StoredConfig
```

Comment:
I don't think there's any reason this actually needs to be inside of the `redistricting` app. I just left it where it was because it was one less thing to change to update all references to it.

I think it would make more sense for `django/publicmapping/redistricting/__ini__.py` to be inside of `django/publicmapping/config/`. I think it would make sense to put `config_settings.py.j2` in there as well (in fact, I have a TODO about that).

Reply:
:+1: for making `StoredConfig` its own module somewhere. I can imagine a universe where making it its own module allows us to work on config validation and wizardry in a more isolated location than in the DB main repo

Reply:
I'm going to hold off on doing anything about this for now.

Reply:
I also agree that splitting this out to its own module would be a nice improvement. Want to add a story for it, so the idea doesn't get lost?

Reply:
I ended up doing most all of the work for this to get languages working (see 4893d20a0b62241339347f9916f4332e55ccf072). I like it a lot better as a package that gets installed. The only remaining work for it would be to actually put the package in its own repo.